enclave_manager/Dockerfile

```diff
@@ -165,65 +157,116 @@ RUN ldconfig \
  && rm -rf /tmp/intel-sgx-ssl \
  && echo "SGX_SSL=/opt/intel/sgxssl" >> /etc/environment
 
-COPY . /project/TrustedComputeFramework
+COPY . /project/avalon
 
 ARG SGX_MODE
 ARG MAKECLEAN
 ARG TCF_DEBUG_BUILD
-ARG DISPLAY
-ARG XAUTHORITY
-
-#TODO need to be removed later
-#install avalon-client sdk python package.
-RUN cd /project/TrustedComputeFramework/sdk \
-   && python setup.py bdist_wheel \
-   && pip3 install dist/*.whl
-
-# Install fabric python sdk
-RUN pip3 install virtualenv \
-   && pip3 install git+https://github.com/hyperledger/fabric-sdk-py.git
 
 # Environment setup
-
-RUN echo "TCF_HOME=/project/TrustedComputeFramework/" >> /etc/environment \
- && if [ ! -z "$http_proxy"  ]; then \
-        echo 'Acquire::http::Proxy "'$http_proxy'";' \
-             >> /etc/apt/apt.conf.d/00proxy; \
-        echo "http_proxy=$http_proxy" >> /etc/wgetrc; \
-        echo "http_proxy=$http_proxy" >> /etc/environment; \
-        echo "HTTP_PROXY=$(echo $http_proxy | sed 's,[a-zA-Z]*://,,')" \
-                  >> /etc/environment; \
-    fi \
- && if [ ! -z "$ftp_proxy"  ];  then \
-        echo 'Acquire::ftp::Proxy "'$ftp_proxy'";' \
-             >> /etc/apt/apt.conf.d/00proxy; \
-        echo "ftp_proxy=$ftp_proxy" >> /etc/wgetrc; \
-        echo "ftp_proxy=$ftp_proxy" >> /etc/environment; \
-    fi \
- && if [ ! -z "$https_proxy" ]; then \
-        echo 'Acquire::https::Proxy "'$https_proxy'";' \
-              >> /etc/apt/apt.conf.d/00proxy; \
-        echo "https_proxy=$https_proxy" >> /etc/wgetrc; \
-        echo "https_proxy=$https_proxy" >> /etc/environment; \
-        echo "HTTPS_PROXY=$(echo $https_proxy | sed 's,[a-zA-Z]*://,,')" \
-             >> /etc/environment; \
-    fi
-
 ENV http_proxy=$http_proxy
 ENV https_proxy=$https_proxy
-ENV TCF_HOME=/project/TrustedComputeFramework
+ENV TCF_HOME=/project/avalon
 ENV SGX_SSL=/opt/intel/sgxssl
 ENV SGX_SDK=/opt/intel/sgxsdk
 ENV PATH=$PATH:$SGX_SDK/bin:$SGX_SDK/bin/x64
 ENV PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$SGX_SDK/pkgconfig
 ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$SGX_SDK/sdk_libs
-# Set display, replace value with [IP address of host]:0
-ENV DISPLAY=:0
+ENV TCF_ENCLAVE_CODE_SIGN_PEM="$TCF_HOME/enclave.pem"
+
+RUN openssl genrsa -3 -out $TCF_HOME/enclave.pem 3072
 
-WORKDIR /project/TrustedComputeFramework/tools/build
+WORKDIR /project/avalon/tools/build
 
 RUN make lint && echo "LINT SUCCESS"
```

Comment:
Remove run lint from enclave manager Dockerfile as currently lint checks all directories in Avalon

Reply:
Ok